packages/hw-transport-node-hid/src/isLedgerDevice.js

```diff
@@ -1,6 +0,0 @@
-// We check the usagePage on Win/OSX, and interface on Linux for get only HID device
-export default d =>
-  (["win32", "darwin"].includes(process.platform)
-    ? d.usagePage === 0xffa0
-    : d.interface === 0) &&
```

Comment:
^ this wasn't a problem on linux? to filter the potential multi devices?

Reply:
My bad.
I restored the filtering.

Reply:
`0xF1D0` is specific to FIDO (👀) U2F which is, indeed, purposefully filtered out so we don't list devices with the `U2F` app open, or a currency app with `Browser Support` set to on.

Reply:
Oh, so I need to disable browser support when using `hw-transport-node-hid`, I see! Sorry, the documentation is a bit scarce and I'm a newbie 😉. Thanks!